src/Osma.Mobile.App/ViewModels/Connections/AcceptInviteViewModel.cs

```diff
@@ -51,36 +53,51 @@ public override Task InitializeAsync(object navigationData)
             return base.InitializeAsync(navigationData);
         }
 
-        private async Task<bool> CreateConnection(IAgentContext context, ConnectionInvitationMessage invite)
+        private async Task<String> CreateConnection(IAgentContext context, ConnectionInvitationMessage invite)
```

Comment:
I would rather we either present the dialog within the scope of this function or we throw an exception and catch it above and use that to generate the dialog?

Reply:
@tplooker changed as suggested. Please have a look!